ratis-server/src/main/java/org/apache/ratis/server/storage/RaftStorageImpl.java

```diff
@@ -91,9 +92,9 @@ public CorruptionPolicy getLogCorruptionPolicy() {
 
   private void format() throws IOException {
     storageDir.clearDirectory();
-    metaFile = new RaftStorageMetadataFileImpl(storageDir.getMetaFile());
-    metaFile.persist(RaftStorageMetadata.getDefault());
-    LOG.info("Storage directory " + storageDir.getRoot() + " has been successfully formatted.");
+    metaFile.set(new RaftStorageMetadataFileImpl(storageDir.getMetaFile()));
+    metaFile.get().persist(RaftStorageMetadata.getDefault());
```

Comment:
After created a class, this becomes a single llne
```java
    metaFile.set(storageDir.getMetaFile()).persist(RaftStorageMetadata.getDefault());
```

Reply:
OK